conf/db/upgrade/V1.9__schema.sql

```diff
@@ -284,4 +284,6 @@ ALTER TABLE ShareableVolumeVmInstanceRefVO ADD CONSTRAINT fkShareableVolumeVmIns
 
 ALTER TABLE `LocalStorageResourceRefVO` DROP INDEX `resourceUuid`;
 ALTER TABLE `LocalStorageResourceRefVO` DROP PRIMARY KEY;
+ALTER TABLE `LocalStorageResourceRefVO` DROP FOREIGN KEY `fkLocalStorageResourceRefVOHostEO`;
```

Comment:
The 1.9 release has been out, and the  DB schema  is readonly.
any fix must go V1.11__schema.sql once it's created.
and please describe the bug, we may have been fixed, you may also check the V1.10 schema